airflow/contrib/hooks/aws_glue_job_hook.py

```diff
@@ -152,7 +152,8 @@ def job_completion(self, job_name=None, run_id=None):
                               .format(run_id, job_run_state))
                 return {'JobRunState': job_run_state, 'JobRunId': run_id}
             else:
-                self.log.info("Polling for AWS Glue Job {} current run state with status {}"
+                self.log.info("Polling for AWS Glue Job %s current run state with status %s", job_name,job_run_state)
+
                               .format(job_name,job_run_state))
```

Comment:
This line is now redundant, and will cause errors.

Reply:
fixed this